docs/reference/release-notes/8.19.1.asciidoc

```diff
@@ -0,0 +1,151 @@
+This caused mismatches between the index and simulation behavior.
+{es-pull}132101[#132101] resolves the issue and ensures that the index’s actual mapping is used when available, preserving consistency between simulation and execution.
+===============
+
+Machine Learning::
++
+.Prevents double-counting of allocations in trained model deployment memory estimation
+[%collapsible]
+===============
+A recent refactor introduced a bug that caused the trained model memory estimation to double-count the number of allocations, leading to inflated memory usage projections.
+{es-pull}131990[#131990] resolves the issue by reverting the change and restoring accurate memory estimation for trained model deployments.
+===============
+
+Mapping::
++
+.Fixes decoding failure for non-ASCII field names in `_ignored_source`
+[%collapsible]
+===============
+A decoding error occurred when field names in `_ignored_source` contained non-ASCII characters. 
+This happened because `String.length()` was used to calculate the byte length of the field name, which only works correctly for ASCII characters.
+{es-pull}132018[#132018] resolves the issue by using the actual byte array length of the encoded field name, ensuring proper decoding regardless of character encoding.
+===============
+
+Search::
++
+.Fixes index sort compatibility for `date_nanos` fields in indices created before 7.14
+[%collapsible]
+===============
+Indices created prior to version 7.14 that used an index sort on a `date_nanos` field could not be opened in more recent versions due to a mismatch in the default `index.sort.missing` value.
+A change in version 7.14 modified the default from `Long.MIN_VALUE` to `0L`, which caused newer versions to reject those older indices.
+{es-pull}132162[#132162] resolves the issue by restoring the expected default value for indices created before 7.14, allowing them to open successfully in newer versions. (issue: {es-issue}132040[#132040])
+===============
++
+.Fix missing removal of query cancellation callback in QueryPhase
+[%collapsible]
+===============
+The timeout cancellation callback registered in `QueryPhase` via `addQueryCancellation` was not removed after the query phase completed.
+This caused unintended timeouts or cancellations during subsequent phases under specific conditions (such as large datasets, low timeouts, and partial search results enabled).
+{es-pull}130279[#130279] resolves the issue and ensures predictable behavior by reintroducing the cleanup logic. (issue: {es-issue}130071[#130071])
+===============
+
+[[enhancement-8.19.1]]
+[float]
+=== Enhancements
+
+Engine::
+* Records available disk space when merge tasks are scheduled, helping to diagnose issues caused by running merges despite low disk space on nodes. This information appears in heap dumps and can inform adjustments to `indices.merge.disk.watermark.high` or inspire future enhancements such as aborting already running merges. {es-pull}131711[#131711]
```

Comment:
Slightly rewording this to put an emphasis on the new change (adding a noun: disk space that is now recorded...)

```suggestion
* Available disk space is now recorded when merge tasks are scheduled, helping to diagnose issues caused by running merges despite low disk space on nodes. This information appears in heap dumps and can inform adjustments to `indices.merge.disk.watermark.high` or inspire future enhancements such as aborting already running merges. {es-pull}131711[#131711]
```